mapping_information/cardano_preprocessing/get_cardano_info_balance.py

```diff
@@ -0,0 +1,127 @@
+import requests
+import pathlib
+import logging
+import json
+import time
+import sys
+
+def get_pool_data():
+    """
+    Queries the Balance Pool Groups API and Cardanoscan to get pool data and write it to a json file.
+    :returns: dictionary, where each key is a pool's hash and the corresponding value is a dictionary with the
+    pool's metadata (name, ticker, homepage, description)
+    :returns: list of dictionaries, where each each dictionary contains metadata (pool_hash, pool_id, pool_group, pool_ticker) 
+    representing a pool.
+    """
+    logging.info("Getting pool data..")
+    filename = 'cardano_pool_data.json'
```

Comment:
That makes sense.  My primary reasons for integrating the Balance data was to validate the EDI consensus tool against known data sourced by the community.  Although the community data doesn't have the rigorously verified legal links, websites, etc., it was heartening to see that the EDI methodology resulted in very similar results as community tools.  For example, the Nakamoto Coefficient reported by Balance by simply counting pool stake is 30, but using the same data with this tool that counts actual historical blocks gave a Nakamoto Coefficient of 32.  I think the differences could likely be explained by the randomness of slot battles or pools missing their propagation windows.

I could save the Balance data to separate files, but some additional options I've considered would be to add a `--balance` flag to the existing `get_cardano_info.py`.  This method could use the existing identifiers/clusters as a source of truth (and retain the websites, links, etc.) and only fill in the gaps that are missing.  The `link` or `source` fields here could be `Balance API`.

Otherwise, I could simply add the Balance data to the existing identifiers/clusters directly to the JSON which is less invasive and in-line with your contribution guidelines.

Do you have a preference for any of these options?  I can get to work on these changes but it may take me some time to implement into the pull request.

Reply:
If you could add them to the existing identifiers / clusters files that would be great! Thanks again for taking the time to contribute :)